src/Features/CSharp/Portable/EditAndContinue/BreakpointSpans.cs

```diff
@@ -801,6 +801,11 @@ internal static TextSpan GetEnvelope(SyntaxNode declaration)
                 return TextSpan.FromBounds(firstSpan.Start, lastSpan.End);
             }
 
+            if (declaration is CompilationUnitSyntax unit && unit.ContainsTopLevelStatements())
+            {
+                return TextSpan.FromBounds(unit.Members[0].SpanStart, unit.Members.OfType<GlobalStatementSyntax>().Last().Span.End);
```

Comment:
Just noticed that local functions are parsed as "GlobalStatementSyntax" (I have no idea whether they should be included here or not).

Reply:
Not sure what you mean by this comment. Yes, any top level statement is contained within a GlobalStatementSyntax, including local functions. This bit of code is returning the span for the source of the top level statements, so includes local functions, just like the span for a regular method would include local functions.